test/resolve-config.test.js

```diff
@@ -21,13 +25,14 @@ test('Returns user config via environment variables', t => {
   const gitlabUrl = 'https://host.com';
   const gitlabApiPathPrefix = '/api/prefix';
   const assets = ['file.js'];
+  const proxy = undefined;
 
   t.deepEqual(
     resolveConfig(
       {assets},
       {env: {GITLAB_TOKEN: gitlabToken, GITLAB_URL: gitlabUrl, GITLAB_PREFIX: gitlabApiPathPrefix}}
     ),
-    {gitlabToken, gitlabUrl, gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix), assets}
+    {gitlabToken, gitlabUrl, gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix), assets, proxy}
```

Comment:
**nitpick:** Since we reformatted the object in the test case below, let's reformat this one to match.

Reply:
I updated proxy value to match {} when proxy is not configured.